tasks/setup.yaml

```
@@ -7,7 +7,7 @@ tasks:
     actions:
       - description: Create k3d cluster with UDS Core Istio
         # renovate: datasource=github-tags depName=defenseunicorns/uds-core versioning=semver
-        cmd: uds deploy oci://defenseunicorns/uds/bundles/k3d-core-istio-dev:0.10.0-${UDS_ARCH} --confirm --no-progress
+        cmd: uds deploy oci://defenseunicorns/uds/bundles/k3d-core-istio-dev:0.12.0-${UDS_ARCH} --confirm --no-progress
```

```suggestion
        cmd: uds deploy oci://defenseunicorns/uds/bundles/k3d-core-istio-dev:0.12.0 --confirm --no-progress
```
Since uds-cli will look up architecture now, we can drop this.